src/components/teamBoxes/TeamOneBox.jsx

```diff
@@ -0,0 +1,164 @@
+    const teamTwo = await get(teamTwoRef)
+    const teamTwoOpsAndSpys = teamTwo.val();
+    let teamTwoSpymaster;
+    let teamTwoOperatives;
+    //Grabbing team twos info
+    if(teamTwoOpsAndSpys && teamTwoOpsAndSpys.spymaster){
+      teamTwoSpymaster = Object.keys(teamTwoOpsAndSpys.spymaster);
+    }
+    //Grabbing team twos info
+    if(teamTwoOpsAndSpys && teamTwoOpsAndSpys.operatives){
+      teamTwoOperatives = Object.keys(teamTwoOpsAndSpys.operatives);
+    }
+    //If a player is on team 2, they cannot join this team
+    if(teamTwoSpymaster && teamTwoSpymaster.includes(playerId) || teamTwoOperatives && teamTwoOperatives.includes(playerId)){
+      console.log('Cannot join the other team!')
+    } else {
+      //Here we want to check if a player is already an operative, so that they cannot join both.
+      await get(teamOneOperativesRef).then((snapshot)=> {
+        //If players already exist as team one operatives:
+        if(snapshot.exists()){
+          //Now we can check if the player is an operative, if they are for now we just console log
+          const teamOneOperatives = Object.keys(snapshot.val());
+          if(teamOneOperatives.includes(playerId)){
+            // later we should probably refactor thisso that something on the UI is triggered
+            console.log('cannot join both the spymasters and the operatives')
+          } else{
+            // if they are not an operative, then we allow them to join as a spymaster
+            set(child(teamOneSpymasterRef, playerId), {playerId, username})
+          }
+        } else {
+          // if the snapshot is null, then no one is a spymaster and we can allow this player to be an operative
+          // this code might be redundant, but I figured it could account for an edge case
+          set(child(teamOneSpymasterRef, playerId), {playerId, username})
+        }
+      })
+      //Dispatch to redux
+      get(teamOneSpymasterRef).then((snapshot)=> {
+        if(snapshot.exists()){
+          dispatch(setTeamOneSpymaster(Object.values(snapshot.val())))
+        }
+      })
+    }
+  }
+  useEffect(()=>{
+    onValue(teamOneRef, (snapshot) => {
+      if(snapshot.exists()){
+        const teamOne = snapshot.val()
+        if(teamOne.operatives){
+          const teamOneOperativesFirebase = Object.values(teamOne.operatives);
+          dispatch(setTeamOneOperatives(teamOneOperativesFirebase))
+        }
+        if(teamOne.spymaster){
+          const teamOneSpymasterFirebase = Object.values(teamOne.spymaster);
+          dispatch(setTeamOneSpymaster(teamOneSpymasterFirebase))
+        }
+      }
+    })
+  }, [])
+  return (
+    <div className="redBoxCard">
+      <div>Team 1</div>
+      <div className="redOpsAndSpys">
+        <div>
+          <p>Operative(s)</p>
+          {
+            teamOneOperatives.map((player, index)=> {
+              return <p
+              key={index}
+              >{player.username}</p>
```

Comment:
maybe at some point we can break players out into their own component but for now this looks great